cluster-stamp-bu0001a0008.json

```diff
@@ -474,19 +474,41 @@
                     {
```

Comment:
templetized same as the spoke. Most probably using the the application instance number.

Reply:
done